doc/boxes.el

```diff
@@ -35,76 +35,72 @@
 ;;          (global-set-key "\C-cr" 'boxes-remove)
 
 ;;; Code:
-(eval-when-compile (require 'cl))
```

Comment:
Deprecated and unnecessary. This was probably added for ‘dolist’, but that has been built in for many years plus I’ve removed ‘dolist’ anyway.